paket.lock

```diff
@@ -1,25 +1,36 @@
-REDIRECTS: ON
 NUGET
-  remote: http://www.nuget.org/api/v2
-  specs:
+  remote: http://api.nuget.org/v3/index.json
     CommandLineParser (1.9.71)
-    FAKE (4.22.8)
-    FSharp.Compiler.Service (2.0.0.6)
+    FAKE (4.36)
+    FSharp.Compiler.Service (3.0)
     FSharp.Core (4.0.0.1)
-    FSharp.Data (2.2.5)
-      Zlib.Portable (>= 1.10) - framework: portable-net40+sl50+wp80+win80
-    FSharpVSPowerTools.Core (2.3)
-      FSharp.Compiler.Service (>= 2.0.0.3)
+    FSharp.Data (2.3.2)
+      Zlib.Portable (>= 1.11) - framework: portable-net45+sl5+win8, portable-net45+win8, portable-net45+win8+wp8+wpa81
+    FSharpVSPowerTools.Core (2.4)
+      FSharp.Compiler.Service (>= 2.0.0.8)
```

Comment:
I wasn't able to build until I upgraded to 3.0. I was hoping that a combination of the latest packages meant the versioning could be updated?